classes/Validate.php

```diff
@@ -640,7 +640,7 @@ public static function isBirthDate($date, $format = 'Y-m-d')
     /**
      * Check for boolean validity.
      *
-     * @param bool|string|null $bool Boolean to validate
+     * @param bool|string|int|null $bool Boolean to validate
```

Comment:
Did you mean mixed type ?
```suggestion
     * @param mixed $bool Value to validate
```